utility/src/main/java/org/oppia/util/firebase/EventLogger.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.util.firebase
```

Comment:
Similar to my other comment--this probably shouldn't be in firebase since it's not necessarily backed by a firebase implementation. Maybe instead: org.oppia.util.logging?

Reply:
Yep, done.